pages/common/trawl.md

```diff
@@ -0,0 +1,19 @@
+# trawl
+
+> Prints out network interface information to the console, much like ifconfig/ipconfig/ip/ifdata.
+
```

Comment:
Could we add a line here for the homepage?

```markdown
> Homepage: <https://github.com/robphoenix/trawl>.

Reply:
I didn't see that in other pages; but LGTM.

Reply:
It's relatively new, so most pages don't have them yet. :+1:

Reply:
I agree with injecting links into the markdown!